scripts/lint.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+ROOT_DIR="${DIR}/.."
+
+python3 -m venv "$ROOT_DIR/.venv"
+source "$ROOT_DIR/.venv/bin/activate"
+
+pip3 install -r "$ROOT_DIR/dev_requirements.txt"
+autopep8 --exit-code --aggressive --recursive --in-place --verbose ./src
```

Comment:
```suggestion
autopep8 --exit-code --aggressive --recursive --in-place --verbose --max-line-length 128 ./src
```
a default of 79 seems a bit harsh

Reply:
🏖️